FWCore/ParameterSet/src/ParameterDescriptionBase.cc

```diff
@@ -120,6 +120,9 @@ namespace edm {
 
   void ParameterDescriptionBase::writeCfi_(
       std::ostream& os, bool optional, bool& startWithComma, int indentation, bool& wroteSomething) const {
+    if (label().empty() or label()[0] == '@') {
```

Comment:
This change was related to `SwitchProducer`, and its use of `@...` parameters to convey information from python to C++, right?

Reply:
Yes